src/server/server.cc

```diff
@@ -1680,6 +1677,64 @@ void Server::AdjustOpenFilesLimit() {
   }
 }
 
+void Server::AdjustWorkerThreads() {
```

Comment:
Do we need some kind of locking ( Or CAS ) here to avoid concurrent `AdjustWorkerThreads`?

Reply:
It's unnecessary since Config::Set is an exclusive command, so it won't have the data race condition.